Services/Simulation/ScriptInterpreter.cs

```diff
@@ -9,10 +9,18 @@ namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Simulation
 {
     public interface IScriptInterpreter
     {
-        Dictionary<string, object> Invoke(
+        /// <summary>Invoke one of the device script files</summary>
+        /// <param name="script">Name of the script</param>
+        /// <param name="context">Context, e.g. current time, device Id, device Model</param>
+        /// <param name="state">Current device sensors state</param>
```

Comment:
nit: not just sensors, the device state can hold simulation state values unrelated to sensors